src/endpoints/transactions/entities/transactions.query.options.ts

```diff
@@ -1,5 +1,8 @@
+import { BadRequestException } from "@nestjs/common";
+
 export class TransactionQueryOptions {
-  private static readonly SIZE_LIMIT: number = 50;
+  private static readonly SCAM_INFO_MAX_SIZE: number = 50;
```

Comment:
this one previous value was 100. Is it the current value intended?

Reply:
yes, this is intentional